docs/documentation/siddhi-4.0.md

```diff
@@ -757,6 +761,87 @@ select e1.roomNo, e2.temp as roomTemp
 having e3 is null
 insert into AlertStream;
 ```
+This query sends an alert when the room temperature reaches the temperature set on the regulator. The pattern matching is reset every time the temperature set on the regulator changes.
+
+**Example 2: Identifying the non-occurence of an expected event**
+ ```sql
+ define stream CustomerStream (customerId string, timestamp long);
+ 
+ from every not CustomerStream for 7 days
+ select *
+ insert into OutputStream;
+ ```
+This query receives information about existing customers of the store from the `CustomerStream` stream. It identifies customers that have not visited the store for the last seven days, and outputs that information to the `OutputStream` stream. A message is generated from the `OutputStream` stream with information for those customers about the discounts that are currently offered at the store.
+
+**Example 3: Detecting the non-occurence of an expected event following another event**
```

Comment:
Examples should come under #### Logical Sequences, and also need to update the syntex